zyte_common_items/util.py

```diff
@@ -100,3 +100,21 @@ def __init__(self, target):
 
     def __call__(self, value):
         return cast_metadata(value, self._target)
+
+
+def wrap_description_into_html(description: str) -> str:
+    r"""Convert plain text into an article HTML.
+
+    The format tries to match clear_html.cleaned_node_to_html().
+
+    >>> wrap_description_into_html('')
+    '<article>\n\n</article>'
+    >>> wrap_description_into_html('foo')
+    '<article>\n\n<p>foo</p>\n\n</article>'
+    >>> wrap_description_into_html('foo\nbar')
+    '<article>\n\n<p>foo</p>\n\n<p>bar</p>\n\n</article>'
+    >>> wrap_description_into_html('foo\n\nbar')
+    '<article>\n\n<p>foo</p>\n\n<p>bar</p>\n\n</article>'
```

Comment:
I think it needs a bit more tests, e.g. a case with more \n's in the text, a case with leading/trailing \n, or a case where some lines are separated by 1 newline, and some are by 2.